client-sdk/go/config/identifier.go

```diff
@@ -14,6 +14,8 @@ func ValidateIdentifier(id string) error {
 		return fmt.Errorf("identifier cannot be empty")
 	case len(id) > 64:
 		return fmt.Errorf("identifier must be less than 64 characters long")
+	case id == "default":
+		return fmt.Errorf("identifier cannot be the string 'default'")
 	case !validID.MatchString(id):
 		return fmt.Errorf("identifier must only contain lower-case letters, numbers and _")
```

Comment:
The error message could also be updated.

Reply:
Fixed.